src/interface/src/app/treatments/direct-impacts/metric-filters/metric-filters.component.scss

```diff
@@ -0,0 +1,31 @@
+.grid-container {
+  display: grid;
+  gap: 16px; // Espacio between metrics
+  grid-template-columns: repeat(1, 1fr); // 1 column by default
+  .grid-item {
+    margin-top: 8px;
+    margin-bottom: 8px;
+  }
+}
+
+@media (min-width: 768px) {
```

Comment:
can you use the queries from `media` ? Feel free to add a new one for a medium size if you need, but lets use the same breakpoints in general

Reply:
Updated, you will see the changes in the next commit.